### ISSUE TITLE
OS 10.13 backup size calculation wrong

### ISSUE THREAD

**Issue by faxe78** [restic/restic#1310]
<!--
## Output of `restic version`

restic 0.7.3
compiled with go1.9 on darwin/amd64

## How did you run restic exactly?

restic -r b2:<BUCKET> backup /Users/xxxx/

## What backend/server/service did you use?

B2

## Expected behavior

show correct backup size

## Actual behavior

show wrong size

## Steps to reproduce the behavior

high sierra with apfs disk.

## Do you have any idea what may have caused this?

apfs has a kind of dedup feature called clones. i tried some online storage services, so i have directories like dropbox, nextcloud and so on with the same data in it

restic tells me my backup would be 1041.154 GiB but my disk has only 500 GB.

## Do you have an idea how to solve the issue?

if dedup works right i am not sure if this is an issue at all, my backup is not finished yet?

**Comment by rawtaz** [comment]
Please report back when the backup is finished, and include the entire restic output in the report, so there's more concrete information to look at. Thanks!

**Comment by faxe78** [comment]
yes dedup kicked in. in general it looks like it is confusing that the calculated backup size is bigger than the actual size of the disk. 

$ restic -r b2:<bucket-name> backup /Users/xxx/
enter password for repository:
scan [/Users/xxx]
scanned 178613 directories, 1071088 files in 1:19

warning and errors about files.

[7:13:31] 100.00%  33.117 MiB/s  841.216 GiB / 841.208 GiB  1250074 / 1249701 items  6 errors  ETA 0:00
duration: 7:13:31, 33.12MiB/s
snapshot 865a89ab saved

**********

actual size of the backup. 

Current Files: 41,301
Current Size: 210.9 GB

**Comment by fd0** [comment]
I agree that this could be better. Restic at the moment is rather dumb concerning file sizes: It won't detect advanced things like deduplicating file systems, and also not (yet) more basic things such as sparse files. But all will be handled just fine by the deduplication algorithm, just probably not in the most efficient way.
